help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,165 @@
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate audiences to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>* To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audiences to this destination.
+
+### Mapping suggestions
+
+Processing requires name and address elements.  Mapping suggestions are provided in the table below listing attributes on your destination side that are used by Acxiom processing that customers can map profile attributes to.  This should be treated as suggestions as not all elements are required and the source values will depend on the needs of the account, but providing as much as possible will aid in matching.
+
+| Target Field | Source Description                                          |
+|--------------|-------------------------------------------------------------|
+| name         | The person.name.fullName value in Experience Platform.      |
+| firstName    | The person.name.firstName value in Experience Platform.     |
+| lastName     | The person.name.lastName value in Experience Platform.      |
+| address1     | The mailingAddress.street1 value in Experience Platform.    |
+| address2     | The mailingAddress.street2 value in Experience Platform.    |
+| city         | The mailingAddress.city value in Experience Platform.       |
+| state        | The mailingAddress.state value in Experience Platform.      |
+| zip          | The mailingAddress.postalCode value in Experience Platform. |
+
+>[!NOTE]
+>
+>Additional fields not listed will be ignored
+
+## Review your dataflow ##
```

Comment:
please remove the `##` at the end of the heading

Reply:
Removed